catalog/spatial/registry/registry-common/src/test/java/org/codice/ddf/registry/common/metacard/TestRegistryObjectMetacardType.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package org.codice.ddf.registry.common.metacard;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.nullValue;
+
+import java.util.Set;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.junit.Test;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.Metacard;
+
+public class TestRegistryObjectMetacardType {
+
+    @Test
+    public void testConstruction() {
+        RegistryObjectMetacardType registryObjectMetacardType = new RegistryObjectMetacardType();
+
+        assertThat(registryObjectMetacardType, not(nullValue()));
+        Set<AttributeDescriptor> descriptors = registryObjectMetacardType.getAttributeDescriptors();
+        assertThat(descriptors, not(nullValue()));
+        assertThat(CollectionUtils.isEmpty(descriptors), is(false));
+
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(Metacard.ID)
+                .isMultiValued(), is(false));
+    }
+
+    @Test
+    public void testMetacardHasBasicDescriptorsAsIsStoredTrue() {
+        RegistryObjectMetacardType registryObjectMetacardType = new RegistryObjectMetacardType();
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(Metacard.TAGS)
```

Comment:
This test might be cleaner and easier to read/modify if you were to create a static list of all these attributeDescriptors, then iterate over it thusly:
```java
for (String attrDesc: attributeDescriptors) {
    assertThat(registryObjectMetacardType.getAttributeDescriptor(attrDesc).isStored(), is(true));
}
```